airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -72,23 +74,27 @@ def get_conn(self):
             num_retries=self.num_retries
         )
 
-    def get_service(self):
+    def get_service(self) -> Any:
         """
         Returns a BigQuery service object.
         """
         http_authorized = self._authorize()
         return build(
             'bigquery', 'v2', http=http_authorized, cache_discovery=False)
 
-    def insert_rows(self, table, rows, target_fields=None, commit_every=1000):
+    def insert_rows(
+        self, table: Any, rows: Any, target_fields: Any = None, commit_every: Any = 1000, replace: Any = False
+    ) -> NoReturn:
         """
         Insertion is currently unsupported. Theoretically, you could use
         BigQuery's streaming API to insert rows into a table, but this hasn't
         been implemented.
         """
         raise NotImplementedError()
 
-    def get_pandas_df(self, sql, parameters=None, dialect=None):
+    def get_pandas_df(
+        self, sql: str, parameters: Union[Iterator, Mapping] = None, dialect: str = None
```

Comment:
It's a good idea too use the same types in docstrings.

Reply:
Did you mean using `iterable` instead of `Iterator`?

Reply:
No. You should update docsrribg also.

Reply:
In this case `Iterable` looks more correct I think. WDYT?

Reply:
Is it resolved?

Reply:
I've put `Iterable` as it seems more valid here (also docstring says `Iterable`).